src/components/Button.stories.js

```diff
@@ -4,6 +4,7 @@ import { DeleteButton } from './Button.js';
 import { EditButton } from './Button.js';
 import { MainNavButton } from './Button.js';
 import { ModalButton } from './Button.js';
+import { QuitButton } from './Button.js';
```

Comment:
You could do all 4 imports at once `import { QuitButton, ModalButton, MainNavButton, EditButton } from './Button.js';`